code/game/objects/items/weapons/grenades/holy_grenade.dm

```diff
@@ -0,0 +1,39 @@
+/obj/item/weapon/grenade/holy
+	name = "holy hand grenade"
+	desc = "One of the many sacred relics made for blowing thine enemies to tiny bits."
+	icon_state = "holy_grenade"
+	det_time = 3.5 SECONDS
+	armsound = 'sound/weapons/vampkiller.ogg'
+	var/activated_at 
+
+/obj/item/weapon/grenade/holy/attackby(obj/item/weapon/W, mob/user)
+	if(W.is_screwdriver(user))
+		to_chat(user, "<span class = 'warning'>There's no timer to change!</span>")
+		return
+	..()
+
+/obj/item/weapon/grenade/holy/activate()
+	activated_at = world.time
+	..()
+
+/obj/item/weapon/grenade/holy/prime()
+	..()
+	playsound(src, 'sound/misc/adminspawn.ogg', 75, 0, 1)
+	sleep(15)
+	for(var/mob/living/carbon/human/H in view(4,get_turf(src)))
+		if(isvampire(H) || iscultist(H))    
+			H.dust()
+	for(var/mob/living/simple_animal/C in view(4,get_turf(src)))
+		if(C.supernatural)
+			C.death()
+	explosion(get_turf(src), 0, 0, 2, 3)
+	qdel(src)
+
+/obj/item/weapon/grenade/holy/throw_at(var/atom/A, throw_range, throw_speed)
+	var/thrown_when = world.time - activated_at
+	if(thrown_when > 3.4 SECONDS || thrown_when < 2.6 SECONDS)
+		to_chat(usr, "<span class='warning'>You didn't count to three.</span>")
```

Comment:
Does `throw_at()` really not have a `user` arg? If not, I guess that's out of the scope of this PR, but I bet this will lead to silly nonsense in weird edge cases.

Reply:
theres no user arg, shurikens do it the same way and just send a message to usr

Reply:
Someone should probably add a `user` arg to it at some point, but not in this PR